router/config.go

```diff
@@ -120,6 +122,7 @@ type Config struct {
 		}
 	}
 	Plugins []string
```

Comment:
Since cost is adjustable via the model we don't need it adjustable via the config file

Reply:
removed!